### ISSUE TITLE
Urls for n_m3u8dl need to be re-edited

### ISSUE THREAD

**Issue by Worz01** [rowrawer/stream-detector#186]
Stream urls with ytdlp and others are fine however for n_m3u8dl-re, the url returns string like "User-Agent: Mozilla%2F5.0%20(Windows%20NT%2010.0%3B%20Win64%3B%20x64%3B%20rv%3A109.0)%20Gecko%2F20100101%20Firefox%2F110.0". This has to be manually edited into the ascii url everytime before cmd reads it, else returns an error. Not a huge problem by itself, but annoying when you are in a hurry.

**Comment by rowrawer** [comment]
I don't think it should return errors, the tool should be able to parse it as is. That being said, as mentioned in https://github.com/rowrawer/stream-detector/issues/179, commands made for N_m3u8DL will indeed no longer be encoded in the next version.

**Comment by Worz01** [comment]
> I don't think it should return errors, the tool should be able to parse it as is. That being said, as mentioned in #179, commands made for N_m3u8DL will indeed no longer be encoded in the next version.

Why? I just joined and Im already heartbroken.

**Comment by rowrawer** [comment]
> Why? I just joined and Im already heartbroken.

You're heartbroken by the issue being fixed?

**Comment by Worz01** [comment]
> I don't think it should return errors, the tool should be able to parse it as is. That being said, as mentioned in #179, commands made for N_m3u8DL will indeed no longer be encoded in the next version.



> > Why? I just joined and Im already heartbroken.
> 
> You're heartbroken by the issue being fixed?

Sorry misunderstood. deleted that. and thanks a lot for making this awesome tool. This was one of the primary reasons I registered.

And yes you are right. For downloads from youtube and moany sites, this does not affect downloads. But Im having issues on some sites that require cookies and some paid sites (iframe mediadelivery).

**Comment by Worz01** [comment]
> No worries. It should be fine in the next update.

Thanks a ton. Any release date finalized yet?

**Comment by rowrawer** [comment]
No, no plans ever, I only work on this addon in my free time when I feel like it. Tomorrow or next year :) I might upload the new version soon, though Mozilla have been taking their time recently with the approval process.

**Comment by Worz01** [comment]
> No, no plans ever, I only work on this addon in my free time when I feel like it. Tomorrow or next year :) I might upload the new version soon, though Mozilla have been taking their time recently with the approval process.

Sure. Thanks a lot for this once again.

**Comment by rowrawer** [comment]
The headers for this tool should finally work properly as of v2.11.2 once it gets approved by Mozilla.
